webpack/assets/javascripts/react_app/components/ReportTemplatesTable/ReportTemplatesTable.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
We created a test helper package which creates snapshots automatically
Basically we have 4 kinds of tests files:
1. unit testing - creates a snapshot from the unconnected component (no redux-store nor actions are involved)
2-3. Reducer and Action testing (Redux part)
4. Integration test - a combination of the three above, testing a component including the state

For more information and some snippet examples, please see [here](https://github.com/theforeman/foreman-js/tree/master/packages/test) or within foreman's storybook